README.md

```diff
@@ -224,9 +224,12 @@ To run the tests behind proxy using Notebook:
    and run the Notebook.
    
    Currently, the following tests are supported to run behind proxy:
+   * e2e-wine
    * katib
-   * kserve
    * kfp_v2
+   * kserve
+   * mlflow-kserve
+   * mlflow-minio
    * training
```

Comment:
if this is all of the tests, can we just remove this section now?

Reply:
We 're waiting for mlflow vanilla test ref #108